composer.json

```diff
@@ -61,7 +61,7 @@
     "ext-dom": "*",
     "oat-sa/lib-test-cat": "2.3.7",
     "oat-sa/oatbox-extension-installer": "~1.1||dev-master",
-    "qtism/qtism": "~0",
+    "qtism/qtism": "^0.24.11",

```

Comment:
Please note this actually potentially prevents another issue of accidentally updating to a new major version, as the previous constraint didn't have an upper bound in the major version (legacy releases use 0.<major>.<minor/patch>` versioning).

Reply:
In fact, this is done in the extension-tao-itemqti. If you do this, you've to do it on both extensions...